inc/opx/private/cps_class_map_query.h

```diff
@@ -43,7 +43,7 @@
 #include <string>
 
 /** @cond HIDDEN_SYMBOLS */
-#define CPS_DEF_PRODUCT_LOC "/opt/dell/os10"
+#define CPS_DEF_PRODUCT_LOC "/opx"
 #define CPS_DEF_SEARCH_PATH CPS_DEF_PRODUCT_LOC "/lib"        //the location of the generated class
 #define CPS_DEF_SEARCH_PATH_CFG "/etc" CPS_DEF_PRODUCT_LOC
 #define CPS_DEF_META_SEARCH_PATHS  CPS_DEF_SEARCH_PATH ":/etc/opx:" CPS_DEF_SEARCH_PATH_CFG
```

Comment:
@ArthiSivanantham, Can you double check this?  I think CPS_DEF_SEARCH_PATH will be /usr/lib, CPS_DEF_SEARCH_PATH_CFG will be /opt/opx.  This suggests that we can't use a common CPS_DEF_PRODUCT_LOC definition.